packages/codegen/templates/schema00/domains/{{domain}}/schema/{{name}}.test.js

```diff
@@ -190,7 +190,7 @@ describe('{{name}}', () => {
                 const [obj, attrs] = await createTest{{name}}(admin)
 
                 const client = await makeClient()
-                await expectToThrowAccessDeniedErrorToObj(async () => {
+                await expectToThrowAuthenticationErrorToObjects(async () => {
```

Comment:
Seems like it's not a part of this PR)

Reply:
Every time after generation new schema, this test fails, because of wrong expectation.
I've fixed it